Client/Frontend/Browser/TabManager.swift

```diff
@@ -164,9 +164,9 @@ class TabManager: NSObject {
         configuration.preferences.javaScriptCanOpenWindowsAutomatically = !Preferences.General.blockPopups.value
         configuration.dataDetectorTypes = .all
         
-//        if !Preferences.General.mediaAutoPlays.value {
-//            configuration.mediaTypesRequiringUserActionForPlayback = .all
-//        }
+        if !Preferences.General.mediaAutoPlays.value {
+            configuration.mediaTypesRequiringUserActionForPlayback = .all
```

Comment:
default seems to be audio only, for our case we can translate to have the preference as enabled by default

```
(lldb) po configuration.mediaTypesRequiringUserActionForPlayback == .all
false

(lldb) po configuration.mediaTypesRequiringUserActionForPlayback == .audio
true
```